### PR TITLE
Task-41235 : Add MFA system step 1 : ask user a static token when he is going on a protected page

### PR DESCRIPTION
**Pull request by rdenarie**
This commit add a feature to ask a token to the user which want to access to a protected page
Protected pages are page from groups navigations configured in properties to be protected
The token is a static token for the moment, waiting for a complete integration in a next step

### CODE REVIEW

File: commons-mfa/src/main/java/org/exoplatform/mfa/api/MfaHandler.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    return true;
+  
+  }
+  
+  protected boolean dispatch(String path, ServletContext context, HttpServletRequest req, HttpServletResponse res)  throws ServletException, IOException {
+    RequestDispatcher dispatcher = context.getRequestDispatcher(path);
+    if (dispatcher != null) {
+      dispatcher.forward(req, res);
+      return true;
+    } else {
```
Comment (azayati) [inline commons-mfa/src/main/java/org/exoplatform/mfa/api/MfaHandler.java]: I think that the else is useless

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: commons-mfa/src/main/java/org/exoplatform/mfa/api/MfaHandler.java (hunk 1 above)
Comment (rdenarie) [reply commons-mfa/src/main/java/org/exoplatform/mfa/api/MfaHandler.java]: done

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**